### ISSUE TITLE
Problem loading Trained Model Checkpoints

### ISSUE THREAD

**Issue by jpsuarezq** [ameraner/dsen2-cr#22]
I had tried to load the .hdf5 trained model checkpoint from DSen2-CR model trained by optimizing the CARL loss and model trained on a plain L1 loss in tensorflow and pytorch but in both cases, the weights are empty. 

Is it a download problem or is the file is corrupt or maybe something else? Can u please check?

**Comment by ameraner** [comment]
Closing as stale
